projects/observability/src/shared/dashboard/data/graphql/specifiers/secondary-entity-specification.model.ts

```diff
@@ -0,0 +1,46 @@
+import { Dictionary } from '@hypertrace/common';
+import { SpecificationModel } from '@hypertrace/distributed-tracing';
+import { Model, ModelProperty, STRING_PROPERTY } from '@hypertrace/hyperdash';
+import { Entity, EntityType } from '../../../../graphql/model/schema/entity';
+import { EntitySpecification } from '../../../../graphql/model/schema/specifications/entity-specification';
+import { SecondaryEntitySpecificationBuilder } from '../../../../graphql/request/builders/specification/entity/secondary-entity-specification-builder';
+
+@Model({
+  type: 'secondary-entity-specification'
+})
+export class SecondaryEntitySpecificationModel extends SpecificationModel<EntitySpecification> {
```

Comment:
Still trying to understand this. how will this be used?

Reply:
This seems identical in purpose to the existing `entity-specification-model` and has the same api (except casing - model's use `kebab-case`). Specs never really have a concept of "primary" or "secondary" since they don't provide the context - they just provide one or more selections under a parent context.

Reply:
Yeah, I can't remember what lead me down the road to this since I've picked up and shelved this PR so many times. Unneeded now though. Removing.